xml/ay_partitioning.xml

```diff
@@ -2510,6 +2511,50 @@ as per ancorgs' comment on Github-->
       </varlistentry>
      </variablelist>
      
+     <para>
+       The <literal>controller_id</literal> element is required.
+     </para>
+     <para>
+       There are two optional elements, <literal>wwpn</literal> 
+       (Worldwide Port Number, the target port through which the SCSI device 
+       is attached), and <literal>fzp_lun</literal>
+       (logical unit number of the SCSI device). It is not necessary to 
+       specify these for FCP devices running in NPIV (Node Port ID 
+       Virtualization) mode, and when the kernel boot parameter 
+       <literal>allow_lun_scan</literal> is set to 1 (the default setting), 
+       which enables automatic LUN scanning by the kernel.
+     </para>
+     <para>
+       If automatic LUN scanning is not available, set the 
+       <literal>wwpn</literal> and <literal>fzp_lun</literal> options 
+       manually.
+     </para>
+
+     <variablelist>
+      <varlistentry>
+        <term>wwpn</term>
+       <listitem>
+       <para>
+        Worldwide port number
+       </para>
+<screen>&lt;wwpn&gt;0x500507630300c562&lt;/wwpn&gt;</screen>
+       </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>fzp_lun</term>
+       <listitem>
+       <para>
+        Logical unit number
+       </para>
+<screen>&lt;fzp_lun&gt;0x4010403200000000&lt;/fzp_lun&gt;</screen>
+       </listitem>
+      </varlistentry>
+     </variablelist>
+     
+     <para>
+       See the IBM documentation for more information,
+       <link xlink:href="https://www.ibm.com/docs/en/linux-on-systems?topic=cd-manually-configured-fcp-luns"/>.
```

Comment:
The URL points to our book for another distro. Also I would like to point to the parent section as that distinguishes automatically scanned LUNs and manually attached LUNs and thus nicely supports the AutoYaST documentation here.
SLES 15: https://www.ibm.com/docs/en/linux-on-systems?topic=wsd-configuring-devices
SLES 12: https://www.ibm.com/docs/en/linux-on-systems?topic=sofc-working-scsi-devices-2

Reply:
Those are much better, thanks!